src/Config/CoreComponent/Config.php

```diff
@@ -37,7 +38,7 @@ public function getQuery(): ?string
     {
         try {
             return $this->getValue(['parameters', 'query']);
```

Comment:
Tady když to upravíš na getStringValue() a analogicky pro další typy, vyřeší ti to většinu těch erorrů v baseline PHPStanu.

Reply:
Nice ještě ten bool tam chybý k dokonalosti. V connection máme https://github.com/keboola/connection/blob/master/Package/Common/src/BoolUtils.php#L12